CRITICAL - 2022-11-22 00:33:33 --> Invalid file: web/header-2.php
in SYSTEMPATH\Exceptions\FrameworkException.php on line 33.
 1 SYSTEMPATH\View\View.php(201): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('web/header-2.php')
 2 SYSTEMPATH\Common.php(1137): CodeIgniter\View\View->render('web/header-2', [], true)
 3 APPPATH\Controllers\Home.php(31): view('web/header-2', [...])
 4 SYSTEMPATH\CodeIgniter.php(904): App\Controllers\Home->news()
 5 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 6 SYSTEMPATH\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2022-11-22 22:00:33 --> Error connecting to the database: mysqli_sql_exception: No connection could be made because the target machine actively refused it.
 in C:\xampp\htdocs\test\da_sein\system\Database\MySQLi\Connection.php:171
Stack trace:
#0 C:\xampp\htdocs\test\da_sein\system\Database\MySQLi\Connection.php(171): mysqli->real_connect('localhost', 'root', '', 'pixel', 3306, '', 0)
#1 C:\xampp\htdocs\test\da_sein\system\Database\BaseConnection.php(382): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\test\da_sein\system\Database\BaseConnection.php(588): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\test\da_sein\system\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\test\da_sein\app\Controllers\Home.php(18): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\test\da_sein\system\CodeIgniter.php(904): App\Controllers\Home->index()
#6 C:\xampp\htdocs\test\da_sein\system\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#7 C:\xampp\htdocs\test\da_sein\system\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\test\da_sein\public\index.php(67): CodeIgniter\CodeIgniter->run()
#9 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No connection could be made because the target machine actively refused it.
 in C:\xampp\htdocs\test\da_sein\system\Database\MySQLi\Connection.php:215
Stack trace:
#0 C:\xampp\htdocs\test\da_sein\system\Database\BaseConnection.php(382): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\test\da_sein\system\Database\BaseConnection.php(588): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\test\da_sein\system\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\test\da_sein\app\Controllers\Home.php(18): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\test\da_sein\system\CodeIgniter.php(904): App\Controllers\Home->index()
#5 C:\xampp\htdocs\test\da_sein\system\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#6 C:\xampp\htdocs\test\da_sein\system\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\test\da_sein\public\index.php(67): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2022-11-22 22:00:33 --> Unable to connect to the database.
Main connection [MySQLi]: No connection could be made because the target machine actively refused it.

in SYSTEMPATH\Database\BaseConnection.php on line 418.
 1 SYSTEMPATH\Database\BaseConnection.php(588): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `slider`', [], false)
 3 APPPATH\Controllers\Home.php(18): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\CodeIgniter.php(904): App\Controllers\Home->index()
 5 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 6 SYSTEMPATH\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2022-11-22 22:30:40 --> Too few arguments to function App\Controllers\Home::work_view(), 0 passed in C:\xampp\htdocs\test\da_sein\system\CodeIgniter.php on line 904 and exactly 1 expected
in APPPATH\Controllers\Home.php on line 57.
 1 SYSTEMPATH\CodeIgniter.php(904): App\Controllers\Home->work_view()
 2 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 3 SYSTEMPATH\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2022-11-22 22:42:09 --> array_count_values(): Can only count STRING and INTEGER values!
in APPPATH\Views\web\work_detail.php on line 56.
 1 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'array_count_values(): Can only count STRING and INTEGER values!', 'APPPATH\\Views\\web\\work_detail.php', 56, [...])
 2 APPPATH\Views\web\work_detail.php(56): array_count_values([...])
 3 SYSTEMPATH\View\View.php(213): include('APPPATH\\Views\\web\\work_detail.php')
 4 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 5 SYSTEMPATH\Common.php(1137): CodeIgniter\View\View->render('web/work_detail', [], true)
 6 APPPATH\Controllers\Home.php(64): view('web/work_detail', [...])
 7 SYSTEMPATH\CodeIgniter.php(904): App\Controllers\Home->work_view('2')
 8 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 9 SYSTEMPATH\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
